context/selection-of-signal-for-I.md

```diff
@@ -0,0 +1,165 @@
+* The same is said from carbon awareness. The way energy consumption is typically calculated (in scope 2 and scope 3) for orgs means it's incredibly hard to prioritise any carbon aware work. Yearly averages mean that if you build in  time shifting in your app it won't reduce the reported scope 2/3 figures for your org which is one of the reasons it's been so challenging to get carbon aware computing onto the agenda for most companies.
+* The GHGP Scope 2 Guidance includes two accounting methods: location-based method and market-based method.
+    * Location-based Accounting: A location-based method reflects the average emissions intensity of grids on which energy consumption occurs. This allows organisations to report their physical account of their carbon emissions as a result of their electricity consumption. 
+    * Market-based Accounting: 
+        * This allows organisations to report lower _gross_ carbon emissions by purchase of energy efforts (market measures) like RECs and GOs.
+            * Before market based reporting organisations had to report carbon emissions from energy consumption as say 10 KT and then separately they could _choose _to neutralise those 10 KT with the purchase of renewable energy certificates (which are financial instruments you can purchase that act like offset to neutralise your energy consumption).
+            * After market based reporting organisations can report carbon emissions from energy consumption as 0 KT (They can report one figure which already takes into account the energy certificates)
+        * Market based reporting has the impact of **deprioritising actions to make software more energy efficient** and **prioritising the purchase of energy certificates**.
+        * With market based reporting organisations now have two ways to report a lower scope 2 emissions figure to the public.(1) Consume less energy by making your processes more efficient. (2) Consume the same energy and purchase certificates. (2) is the easier solution for an organisation to prioritise than (1) which takes significant investment and effort. 
+        * Renewable energy purchasing is often managed by a different team than the teams that develop software, which means organisations can reduce their emissions through market based measures without engaging in any efficiency improvement activity in the software products.
+        * In addition if an organisation is already reporting 0 KT in terms of Scope 2 because of their purchase of energy certificates, and did more work to make their processes more energy efficient, they would still report 0 KT, a metric that stays at 0 regardless of what you do, does not encourage action..
+        * Market based reporting doesn’t incentivize making applications more energy efficient.

```

Comment:
The discussion seems to be around which signals should be allowed in the SCI. In the interest of a more concise and clear document, I believe it would be helpful to remove / shorten content that is not relevant to the discussion. This is a large chunk that creates a lot of noise which can be shortened, or completely removed. 

``` suggestion
```

Reply:
I think we should keep this, one of the key aspects of why this spec exists.

Reply:
I also vote for keeping, this is a key point for a lot of people who are only familiar with carbon reporting and the GHG protocol